### PR TITLE
OAS-5162/support for comments in ip allowlists

### PR DESCRIPTION
**Pull request by nerpaula**

### CODE REVIEW

File: oasis/projects.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ You can create an allowlist as part of
    - A name or the __eye__ icon in the __Actions__ column to view the allowlist.
    - The __pencil__ icon to edit the allowlist.
      You can also view the allowlist and click the __Edit__ button.
+   - Add single-line comments in the __Example CIDR ranges__ section. 
+     Everything after `\\` or `#` is considered a comment until the end of the line.
```
Comment (Robert-Stam) [inline oasis/projects.md:84]: \\\\ --> //

**Review by Robert-Stam — COMMENTED** [review]
See comments

File: oasis/projects.md [hunk 2]
```diff
@@ -80,6 +80,8 @@ You can create an allowlist as part of
    - A name or the __eye__ icon in the __Actions__ column to view the allowlist.
    - The __pencil__ icon to edit the allowlist.
      You can also view the allowlist and click the __Edit__ button.
+   - Add single-line comments in the __Example CIDR ranges__ section.
```
Comment (Robert-Stam) [inline oasis/projects.md:83]: Example CIDR ranges --> Allowed CIDR ranges (1 per line)
 
 
image

**Review by ansoboleva — APPROVED** [review]
reviewed

**Pull request merged**